rofi-power-menu

```diff
@@ -45,9 +47,10 @@ actions[suspend]="systemctl suspend"
 actions[hibernate]="systemctl hibernate"
 actions[reboot]="systemctl reboot"
 actions[shutdown]="systemctl poweroff"
+actions[windows]="systemctl reboot --boot-loader-entry=auto-windows --boot-loader-menu=0.5"
```

Comment:
Is this a generic way to reboot to Windows?

Also, why doesn't normal reboot work? I suppose you can choose which OS to launch during the boot time, right?

Reply:
This PR is quite old. I learned a lot since opening this, so I can explain more what would be the right way to do things.

> Is this a generic way to reboot to Windows?

As noted above by Spaxly, this `systemctl reboot --boot-loader...` command only works when you use both systemd (as init) and systemd-boot (as bootloader).

Because setting the next entry is bootloader specific, one would need to add special commands for each bootloader. As I can see it, grub also supports rebooting into another entry. See the [manpage](https://manpages.ubuntu.com/manpages/xenial/man8/grub-reboot.8.html). You can specify an entry there.

Using another init (i.e. no systemd) completely breaks the entire script, as there is no `systemctl` when using other inits. This is also discussed in #11.

Covering all the cases would be hard I guess, so a decision needs to be taken on which inits and boot loaders should be supported. I guess we could also just start with some inits and bootloaders and then extend if people are interested.

> Also, why doesn't normal reboot work? I suppose you can choose which OS to launch during the boot time, right?

Yes, you are right. Usually when you use dual booting, your bootloader shows a screen where you can select the boot entry for a few seconds. What this command does though, is that it preselects the chosen entry. No user intervention is needed until the other entry/OS is booted up. That is somewhat convenient.